packages/anchors/src/OpenVAnchor.ts

```diff
@@ -0,0 +1,454 @@
+import { BigNumber, BigNumberish, ContractTransaction, ethers } from 'ethers';
+import { OpenVAnchor as OpenVAnchorContract, OpenVAnchor__factory } from '@webb-tools/contracts';
+import { solidityPack } from 'ethers/lib/utils';
+import {
+  toHex,
+  toFixedHex,
+  Utxo,
+  MerkleTree,
+  median,
+  mean,
+  max,
+  min,
+  CircomProvingManager,
+  MerkleProof,
+} from '@webb-tools/sdk-core';
+import { u8aToHex, getChainIdType, ZkComponents } from '@webb-tools/utils';
+import { IAnchor } from '@webb-tools/interfaces';
+
+const zeroAddress = '0x0000000000000000000000000000000000000000';
+function checkNativeAddress(tokenAddress: string): boolean {
+  if (tokenAddress === zeroAddress || tokenAddress === '0') {
+    return true;
+  }
+  return false;
+}
+function sha3Hash(left: BigNumberish, right: BigNumberish) {
```

Comment:
Is there a way we can cleanup the sha3Hash function here? Is it actually using sha3?